### PR TITLE
Update packages to latest for highlighted security vulnerabilities

### PR DESCRIPTION
**Pull request by LisaC-ResDiary**

### CODE REVIEW

File: src/DatabaseVersion.Console/dbversion.Console.csproj [hunk 1]
```diff
@@ -11,6 +11,22 @@
...
+    <UpdateInterval>7</UpdateInterval>
+    <UpdateIntervalUnits>Days</UpdateIntervalUnits>
+    <UpdatePeriodically>false</UpdatePeriodically>
+    <UpdateRequired>false</UpdateRequired>
+    <MapFileExtensions>true</MapFileExtensions>
+    <ApplicationRevision>2</ApplicationRevision>
+    <ApplicationVersion>1.0.0.%2a</ApplicationVersion>
+    <UseApplicationTrust>false</UseApplicationTrust>
+    <PublishWizardCompleted>true</PublishWizardCompleted>
+    <BootstrapperEnabled>true</BootstrapperEnabled>
```
Comment (DavidHaughtonResDiary) [inline src/DatabaseVersion.Console/dbversion.Console.csproj:29]: What is all this stuff?

File: src/DatabaseVersion.Console/dbversion.Console.csproj [hunk 2]
```diff
@@ -32,6 +48,18 @@
...
+  </PropertyGroup>
+  <PropertyGroup>
+    <ManifestKeyFile>dbversion.Console_TemporaryKey.pfx</ManifestKeyFile>
+  </PropertyGroup>
+  <PropertyGroup>
+    <GenerateManifests>true</GenerateManifests>
+  </PropertyGroup>
+  <PropertyGroup>
+    <SignManifests>true</SignManifests>
+  </PropertyGroup>
```
Comment (DavidHaughtonResDiary) [inline src/DatabaseVersion.Console/dbversion.Console.csproj:62]: And this? Looks unrelated to packages 😕

File: src/DatabaseVersion.Console/dbversion.Console_TemporaryKey.pfx
Comment (DavidHaughtonResDiary) [inline src/DatabaseVersion.Console/dbversion.Console_TemporaryKey.pfx:1]: Again not sure what this is, but do we want to commit a temporary key?

**Review by DavidHaughtonResDiary — COMMENTED** [review]
Im quite confused with whats being added here - we've added 270k lines which seems like a lot. What are all these large XML files?

If you look at the security report it seems to the 2 vulnerabilities are still present in the base branch (master) even with these changes.

File: src/DatabaseVersion.Console/dbversion.Console_TemporaryKey.pfx
Comment (LisaC-ResDiary) [reply src/DatabaseVersion.Console/dbversion.Console_TemporaryKey.pfx:1]: I wonder if this is due to be doing a publish? Will try again without publishing to see what happens...
